<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>qt5-qtbase.git - qt5-qtbase</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/qt5-qtbase.git/atom/qtbase-opensource-src-5.5.x-big-endian.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 4 && tokens[4] != '') {
    repo = tokens[4].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='qt5-qtbase.git' href='/cgit/qt5-qtbase.git/'>qt5-qtbase.git</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='el6'>el6</option>
<option value='epel7'>epel7</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f22-5.4.0'>f22-5.4.0</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>qt5-qtbase</td><td class='sub right'>Jon Ciesla</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/qt5-qtbase.git/'>summary</a><a href='/cgit/qt5-qtbase.git/refs/'>refs</a><a href='/cgit/qt5-qtbase.git/log/qtbase-opensource-src-5.5.x-big-endian.patch'>log</a><a class='active' href='/cgit/qt5-qtbase.git/tree/qtbase-opensource-src-5.5.x-big-endian.patch'>tree</a><a href='/cgit/qt5-qtbase.git/commit/qtbase-opensource-src-5.5.x-big-endian.patch'>commit</a><a href='/cgit/qt5-qtbase.git/diff/qtbase-opensource-src-5.5.x-big-endian.patch'>diff</a><a href='/cgit/qt5-qtbase.git/stats/qtbase-opensource-src-5.5.x-big-endian.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/qt5-qtbase.git/log/qtbase-opensource-src-5.5.x-big-endian.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/qt5-qtbase.git/tree/'>root</a>/<a href='/cgit/qt5-qtbase.git/tree/qtbase-opensource-src-5.5.x-big-endian.patch'>qtbase-opensource-src-5.5.x-big-endian.patch</a></div><div class='content'>blob: 619fed3493fe540471946da288c72c5e44c7fc83 (<a href='/cgit/qt5-qtbase.git/plain/qtbase-opensource-src-5.5.x-big-endian.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
</pre></td>
<td class='lines'><pre><code>diff -up qtbase-opensource-src-5.5.0/src/gui/image/qimage_conversions.cpp.than qtbase-opensource-src-5.5.0/src/gui/image/qimage_conversions.cpp
--- qtbase-opensource-src-5.5.0/src/gui/image/qimage_conversions.cpp.than	2015-07-09 17:01:29.605707305 +0200
+++ qtbase-opensource-src-5.5.0/src/gui/image/qimage_conversions.cpp	2015-07-10 12:39:04.022599044 +0200
@@ -1103,7 +1103,7 @@ static bool mask_alpha_converter_rgbx_in
         }
         rgb_data += pad;
     }
-    data-&gt;format = DestFormat;
+    data-&gt;format = QImage::Format_RGBX8888;
     return true;
 #endif
 }
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2015-07-14 16:54:31 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
